Automattic/wp-calypso
# composite-checkout: Refactor CheckoutSummary to CheckoutSummaryArea/Card

#### Changes proposed in this Pull Request

This refactors `CheckoutSummary` (the sidebar) so that it can contain more than one separate card.

Previously, composite-checkout exported `CheckoutSummary`, a bordered card wrapper for sidebar-like content. However, this doesn't give us enough flexibility to add a second card hovered below the summary card (eg: an upsell). 

This PR modifies composite-checkout to export two components instead: `CheckoutSummaryArea` and `CheckoutSummaryCard`; the former is just the floated area with no border or background and the latter is the borders and background. This way it's possible to use them separately to achieve a design with multiple cards.

#### Testing instructions

This should have no effect on checkout and is split off from #41456.

Visit composite checkout to be sure that the summary (the sidebar at wide window widths) is displayed as it was before (notably with a border and white background).
